src/lib/telemetry.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import Glean from '@mozilla/glean/web';
+
+const APP_NAME = "debug-ping-view";
+const APP_TELEMETRY_PREFERENCE_STORAGE_KEY = "telemetryEnabled";
+
+function isTelemetryEnabled() {
```

Comment:
Can we give these functions jsdoc comments? For example:

```js
/**
 * Use the browser `doNotTrack` flag to determine if we collect telemetry.
 *
 * @returns {boolean} True if we will collect telemetry, false otherwise.
 */
function isTelemetryEnabled() {
  return navigator.doNotTrack !== "1";
}
```

These aren't required or anything, but it makes working with this repo a lot easier if every function has a nice description of what is does, especially since most of this codebase uses tech that most aren't super familiar with.

Reply:
Done.